src/frontend/components/SøknadsSteg/Utvidet-DinLivssituasjon/types-and-utilities.ts

```diff
@@ -0,0 +1,19 @@
+export enum Årsak {
```

Comment:
Kanskje denne kan renames til ÅrsakSpråkId?

Reply:
Eller når jeg leser nedover gir det mest mening å ha en Årsak-enum. Kanskje man kan dele opp slik at enum er enum, og en egen mapper til språkId'ene?

Reply:
Høres ut som en god idé, prøver det

Reply:
ble dette ok tru? se toÅrsakSpråkId

Reply:
Ja, supert 🙌